src/stylesheets/components/global/addButton.scss

```diff
@@ -0,0 +1,16 @@
+.add-button-container {
+  height: 40px;
+  width: 100%;
+  min-width: 80px;
+  background: $text-input-bg;
+  background: $grey-button-bg;
+  border-radius: 10px;
+  margin: 20px 0px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.add-button-container:hover {
+  cursor: pointer;
+}
```

Comment:
uh, are new line characters at the end of the file not enforced by linters?

Reply:
The rest looks good! ^ up to you to add a new line character or not. Cheers

Reply:
linters doesn't seem to have an issue with it, so its probably fine

Reply:
added them